custom_components/tahoma/sensor.py

```diff
@@ -107,10 +125,67 @@ async def async_setup_entry(hass, entry, async_add_entities):
     entities = [
         TahomaSensor(device.deviceurl, coordinator)
         for device in data["entities"].get(SENSOR)
-        if device.states
+        if device.states and "BATTERY_" not in device.label
+    ]
+
+    battery_entities = [
+        TahomaBatterySensor(device.deviceurl, coordinator)
+        for device in data["entities"].get(SENSOR)
+        if device.states and "BATTERY_" in device.label
     ]
 
-    async_add_entities(entities)
+    async_add_entities(entities + battery_entities)
+
+
+class TahomaBatterySensor(TahomaDevice):
```

Comment:
Is it also possible to add this to the current TahomaSensor? This PR is a great start for https://github.com/iMicknl/ha-tahoma/issues/223 and it would be good to understand if we can figure out a good way to allow multiple sensors.

Reply:
Not sure I understand what you mean here.

Reply:
You introduced `TahomaBatterySensor`, where we already have a `TahomaSensor` class. This class currently handles all sensors, with different states / values supported. Perhaps we can figure out a way to reuse TahomaSensor, but dynamically assign multiple sensors based on available states.

Reply:
However, we can also do this in a next revision for sure. It would be great if we all can think of a way where this PR can be the base for #223.